OpenRA.Mods.Common/ServerTraits/PlayerPinger.cs

```diff
@@ -36,7 +36,7 @@ public void Tick(S server)
 				lastPing = Game.RunTime;
 
 				// Ignore client timeout in singleplayer games to make debugging easier
-				if (server.LobbyInfo.IsSinglePlayer && !server.Dedicated)
+				if (server.LobbyInfo.NonBotClients.Count() < 2 && !server.Dedicated)
```

Comment:
Is it even possible to have less then one non-bot client? (Otherwise `NonBotClient.Count() == 1` would work as well, wouldn't it?) Anyway, just wondering, this shouldn't make any difference in practice.